extensions/wikia/UserProfilePageV3/UserProfilePage.i18n.php

```diff
@@ -57,6 +57,7 @@
 	'user-identity-box-group-voldev' => 'Volunteer Developer',
 	'user-identity-box-group-threadmoderator' => 'Discussions Moderator',
 	'user-identity-box-group-vanguard' => 'Vanguard',
+	'user-identity-box-group-content-moderator' => 'Content Moderator',
 	'user-identity-box-zero-state-location' => 'Location',
 	'user-identity-box-zero-state-birthday' => 'Birthday',
 	'user-identity-box-zero-state-gender' => 'Gender',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "user-identity-box-zero-state-gender" 35 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)